### PR TITLE
Update jQuery to vanilla js and fix carouselScroll() function

### PR DESCRIPTION
**Pull request by sabinaismailova**

### CODE REVIEW

File: js/custom.js
Comment (smrohrer) [inline js/custom.js:1]: There's a mix of spaces and tabs in this file, due to different contributors using different settings - can we format the entire file using either one or the other?

**Review by smrohrer — COMMENTED** [review]
Good work! Would be great to fix the indentation in this file.

File: js/custom.js
Comment (sabinaismailova) [reply js/custom.js:1]: Yes, I can reformat the file. I'm not sure how to move along with this. Do I make a new pull-request to reformat the file?

Comment (sabinaismailova) [reply js/custom.js:1]: I reformatted the file in the new branch called format-custom-js and made a PR format-custom-js to the carouselScroll branch.

Comment (smrohrer) [reply js/custom.js:1]: Looks good to me, thank you for handling that! You could also commit the changes directly to this branch

Comment (sabinaismailova) [reply js/custom.js:1]: I'll keep that in mind for next time. Thank you!

**Review by bogreudell — APPROVED** [review]

**Pull request merged**